opensearch/2.11/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Repo name: arynai/sycamore-opensearch
+
+ARG TAG=2.11.0
```

Comment:
Isn't `2.11.1` the latest 2.11 release?

Reply:
I guess. This is just a copy of what was previously in the quickstart